FieldOpt/Hdf5SummaryReader/hdf5_summary_reader.cpp

```diff
@@ -8,27 +8,176 @@ Hdf5SummaryReader::Hdf5SummaryReader(const std::string file_path)
   DATASET_NAME_TIMES("TIMES"),
   GROUP_NAME_FLOW_TRANSPORT("FLOW_TRANSPORT"),
   DATASET_NAME_ACTIVE_CELLS("ACTIVE_CELLS"),
-  DATASET_NAME_WELL_STATES("WELL_STATES")
+  DATASET_NAME_WELL_STATES("WELL_STATES"),
+  DATASET_NAME_PRESSURE("PTZ"),
+  DATASET_NAME_SATURATION("GRIDPROPTIME")
 {
     readTimeVector(file_path);
-    readActiveCells(file_path);
     readWellStates(file_path);
+    readActiveCells(file_path);
```

Comment:
I agree, this is likely to reduce performance as the summaryreader is called after each sim for obj.f. calculation, and cell data is only really wanted at the end of the whole run, for postprocessing.